swedish-oidc-claims-specification.md

```diff
@@ -390,104 +388,156 @@ There are basically two ways for a Relying Party to indicate which claims it wis
-delivered, but it is more useful to define more fine grained scopes. Also, for the sake of privacy a relying party should not
-ask for more claims than it actually requires. Therefore, this specification defines a set of scopes that combines standard 
-claims with the claims defined in this specification.
-
-This specification specifically takes into consideration the Swedish eID solutions, and the attributes that are tied to a 
-Swedish eID.
-
-For each scope defined below a set of claims is declared. Each declared claim has an indicator telling whether it is 
-Mandatory or Optional within the given scope. A profile specification extending this attribute specification MAY change 
-a declared claim's requirement from Optional to Mandatory, but MUST NOT lower the requirements from Mandatory to Optional.
-In those cases the extending specification need to define a new scope. Also, a profile specification extending this
-attribute MAY declare new claims, but MUST NOT remove any claims from the scope definition.
-
-<a name="natural-person-name-information"></a>
-### 3.1. Natural Person Name Information
-
-**Scope:** `https://id.oidc.se/scope/naturalPersonName`
-
-**Description:** A scope that defines a claim set that provides basic natural person information without revealing the
-civic registration number of the subject.
-
-| Claim | Description/comment | Reference | Requirement |
-| :--- | :--- | :--- | :--- |
-| `family_name` | Surname/family name | \[[OpenID.Core](#openid-core)\] | Mandatory |
-| `given_name` | Given name | \[[OpenID.Core](#openid-core)\] | Mandatory |
-| `name` | Display name | \[[OpenID.Core](#openid-core)\] | Mandatory |
+This specification defines a number of scopes that are useful within the context of Swedish
+eID usage. 
+
+Many Relying Parties that uses OpenID Connect to authenticate users can not solely depend
+on the user's session at the OpenID Provider and the `sub` claim to log in the user to the
+RP application. In the context of Swedish eID there are some obvious claims that are regarded
+to be "primary" identity claims by Relying Parties, for example a Swedish personal identity
+number. Such claims are needed by the Relying Party in order to log in a user to its application.
+Therefore, this specification's scope definitions will define that some claims are to be delivered
+in the ID token so that a Relying Party can fully log in a user without having to make a,
+potentially, unnecessary call to the UserInfo endpoint.
+
+For each scope defined below, a listing of the claims that the scope value requests (access to) is
+declared. Each scope definition also presents a "claims parameter equivalent", i.e., how the
+claims would be requested using the `claims` request parameter. This tells where claims should be
+delivered (ID token and/or UserInfo endpoint), and whether the claims should be regarded as essential
+or voluntary to deliver by the OP.
+
```

Comment:
Suggest reword to:
"This tells where claims must be delivered as response to the requested scope"

I believe we need a deterministic delivery if just the scope is requested.